<?xml version="1.0" encoding="UTF-8"?>
<integrity name="Integrity JUnit Testing" timestamp="06.03.15 09:25" isotimestamp="2015-03-06T09:25:04" duration="6.545" abortMessage="This is an abort exception in a call">
  <variables />
  <suite id="0" name="integrity.basic.exceptions.abortExceptions.abortExceptionInCall" timestamp="06.03.15 09:25:04.0615">
    <setup />
    <variables />
    <statements>
      <call id="1" line="69" name="noOp" description="Do absolutely nothing." fixture="de.gebit.integrity.tests.fixtures.basic.NoOpFixture#noOp" timestamp="06.03.15 09:25:04.0615">
        <parameters />
        <result duration="0.034" type="success" />
      </call>
      <call id="2" line="70" name="throwAbortException" description="Throws an abortion exception" fixture="de.gebit.integrity.tests.fixtures.basic.NoOpFixture#throwAbortException" timestamp="06.03.15 09:25:04.0631">
        <parameters>
          <parameter name="message" value="This is an abort exception in a call" />
        </parameters>
        <result duration="0.813" type="exception" exceptionMessage="This is an abort exception in a call" exceptionTrace="de.gebit.integrity.exceptions.AbortExecutionException: This is an abort exception in a call&#xD;&#xA;&#x9;at de.gebit.integrity.tests.fixtures.basic.NoOpFixture.throwAbortException(NoOpFixture.java:181)&#xD;&#xA;&#x9;at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#xD;&#xA;&#x9;at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)&#xD;&#xA;&#x9;at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)&#xD;&#xA;&#x9;at java.lang.reflect.Method.invoke(Method.java:597)&#xD;&#xA;&#x9;at de.gebit.integrity.fixtures.FixtureWrapper.execute(FixtureWrapper.java:283)&#xD;&#xA;&#x9;at de.gebit.integrity.runner.DefaultTestRunner.executeCallSingle(DefaultTestRunner.java:1582)&#xD;&#xA;&#x9;at de.gebit.integrity.runner.DefaultTestRunner.executeCall(DefaultTestRunner.java:1527)&#xD;&#xA;&#x9;at de.gebit.integrity.runner.DefaultTestRunner.executeSuite(DefaultTestRunner.java:961)&#xD;&#xA;&#x9;at de.gebit.integrity.runner.DefaultTestRunner.callSuiteSingle(DefaultTestRunner.java:757)&#xD;&#xA;&#x9;at de.gebit.integrity.runner.DefaultTestRunner.runInternal(DefaultTestRunner.java:608)&#xD;&#xA;&#x9;at de.gebit.integrity.runner.DefaultTestRunner.run(DefaultTestRunner.java:498)&#xD;&#xA;&#x9;at de.gebit.integrity.runner.DefaultTestRunner.run(DefaultTestRunner.java:433)&#xD;&#xA;&#x9;at de.gebit.integrity.tests.junit.IntegrityJUnitTest.executeIntegritySuite(IntegrityJUnitTest.java:99)&#xD;&#xA;&#x9;at de.gebit.integrity.tests.junit.basic.exceptions.AbortExceptions.abortExceptionInCall(AbortExceptions.java:127)&#xD;&#xA;&#x9;at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#xD;&#xA;&#x9;at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)&#xD;&#xA;&#x9;at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)&#xD;&#xA;&#x9;at java.lang.reflect.Method.invoke(Method.java:597)&#xD;&#xA;&#x9;at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)&#xD;&#xA;&#x9;at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#xD;&#xA;&#x9;at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)&#xD;&#xA;&#x9;at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#xD;&#xA;&#x9;at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)&#xD;&#xA;&#x9;at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)&#xD;&#xA;&#x9;at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)&#xD;&#xA;&#x9;at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)&#xD;&#xA;&#x9;at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)&#xD;&#xA;&#x9;at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)&#xD;&#xA;&#x9;at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)&#xD;&#xA;&#x9;at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)&#xD;&#xA;&#x9;at org.junit.runners.ParentRunner.run(ParentRunner.java:309)&#xD;&#xA;&#x9;at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#xD;&#xA;&#x9;at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#xD;&#xA;&#x9;at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#xD;&#xA;&#x9;at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#xD;&#xA;&#x9;at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#xD;&#xA;&#x9;at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#xD;&#xA;" />
      </call>
    </statements>
    <teardown />
    <result duration="5.719" successCount="0" failureCount="0" exceptionCount="1" testExceptionCount="0" callExceptionCount="1" />
  </suite>
</integrity>

